### PR TITLE
Fix for issue #184

### PR DESCRIPTION
**Pull request by DeBiese**
I also ran into the translation issue described in issue #184 .

I found the issue at hand.
My fix offers 2 options. 1 is adding an observe on the wzTitle and change the title when wzTitle changes. The other option is to just not use title.

I'm not sure wy title is added on the steps, a quick test learned me that both options work. But in the event that title is used somewhere that I don't know of, option1 is the safest way to go.

### CODE REVIEW

File: src/step.js [hunk 1]
```diff
@@ -15,8 +15,14 @@ angular.module('mgo-angular-wizard').directive('wzStep', function() {
...
           return attributes.template || "step.html";
         },
-        link: function($scope, $element, $attrs, wizard) {
+        link: function ($scope, $element, $attrs, wizard) {
+            //OPTION1: ADD observe on the wzTitle attribute and change the title accordingly
+            $attrs.$observe('wzTitle', function (value) {
+                $scope.title = $scope.wzTitle;
+            });
             $scope.title = $scope.wzTitle;
+            //OPTION2:  DON'T set the title on the scope
```
Comment (jacobscarter) [inline src/step.js]: please remove the comments from `step.js` and I will merge this

**Comment by DeBiese** [comment]
I assumed you opted for option 2, so I removed the line (and all of the comments and the observe on wzTitle).

**Comment by jacobscarter** [comment]
Sorry that's my fault, I should have specified, there are many section of the code that reference and depend on the steps title.  

https://github.com/DeBiese/angular-wizard/blob/c8912b491a607ad63ac486069f9026a49dc8a041/dist/angular-wizard.js#L106
https://github.com/DeBiese/angular-wizard/blob/c8912b491a607ad63ac486069f9026a49dc8a041/dist/angular-wizard.js#L294
wizard HTML template
https://github.com/DeBiese/angular-wizard/blob/c8912b491a607ad63ac486069f9026a49dc8a041/dist/angular-wizard.js#L120

Sorry to make you undo it but option 1 please!

Thank you.

**Comment by DeBiese** [comment]
No problem. I can undo this and implement option 1.

But before I do that, may I point out that the lines you are referring to, to point out that the title is used, all use wzTitle and not title. 

I didn't find any use of the title field on the step other than in the html where there is a bind to step.title or step.wzTitle:
[https://github.com/DeBiese/angular-wizard/blob/c8912b491a607ad63ac486069f9026a49dc8a041/dist/angular-wizard.js#L21](https://github.com/DeBiese/angular-wizard/blob/c8912b491a607ad63ac486069f9026a49dc8a041/dist/angular-wizard.js#L21)

This line together with the line I removed by choosing for option 2 makes that translations don't work currently. Because the title is set once and although the wzTitle changes, the title field never changes and in html step.title is known and used over step.wzTitle every time.

**So what I'm saying is comes down to this**: 
sure, I can undo and add the $observer on wzTitle to update the value of title. But do you know of an actual use of that title field other than in the html that I referenced?

**Pull request merged**